vm/src/sysmodule.rs

```diff
@@ -54,17 +54,26 @@ pub fn make_module(vm: &VirtualMachine, module: PyObjectRef, builtins: PyObjectR
         "cache_tag" => ctx.new_str("rustpython-01".to_string()),
     });
 
-    let path_list = match env::var_os("PYTHONPATH") {
-        Some(paths) => env::split_paths(&paths)
+    let path_list = if cfg!(target_arch = "wasm32") {
```

Comment:
Off topic:

Still I believe that having specific platform checks in the rustpython core library is not correct. We can figure out a way later I believe in which we might have ways to insert paths into the vm. Another point in which there is wasm related code in the virtual machine is the wasm_id. I think the rustpython library must be agnostic of wasm, and we can achieve this.